app/survey_config/survey_config.py

```diff
@@ -71,14 +71,7 @@ def get_footer_legal_links(  # pylint: disable=unused-argument, no-self-use
     ) -> Optional[list[dict]]:
         return None
 
-    def get_data_layer(self, tx_id: Optional[str] = None) -> list[dict]:
-        data_layer = [{"tx_id": tx_id}] if tx_id else []
-        if self.schema:
-            data_layer.append(
-                {
-                    key: self.schema.json[key]
-                    for key in ["form_type", "survey_id", "title"]
-                    if key in self.schema.json
-                }
-            )
-        return data_layer
+    def get_additional_data_layer_context(  # pylint: disable=no-self-use
+        self,
+    ) -> list[dict]:
```

Comment:
I would just have this as list since only the child class method adds a dict?
```suggestion
    ) -> list:
```

Reply:
It does here but when the census equivalent is used it returns a list of dicts so I thought it would be confusing if they had different return types?

Reply:
I understand your argument though because the actual function does just return an empty list

Reply:
Not sure, I don't think it's confusing so long as the type hint for the census config is correct, as anything that uses this parent class would only ever return an empty list. Won't block it on this anyway

Reply:
Ok I'm happy to make the change